UPGRADE_GUIDE.md

```diff
@@ -35,6 +35,24 @@ app.ModuleManager.SetOrderPreBlockers(
 
 ## Adding ProtocolPool Module **OPTIONAL**
 
+:::warning
+
+Using `protocolpool` will cause the following `x/distribution` handlers to return an error:
```

Comment:
```suggestion
Using `protocolpool` will cause the following `x/distribution` handlers to return an error:
```

We should just say "using an external community pool"  because users can add whatever kind of community pool module they want - not just `x/protocolpool`

Reply:
reworded it to suggest external pools may be used here